### PR TITLE
Update google URL and avoid redirecting.

### PR DESCRIPTION
**Pull request by Arman-Hosseini**

### CODE REVIEW

**Review by bocharsky-bw — APPROVED** [review]

File: web/robots.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # www.robotstxt.org/
-# www.google.com/support/webmasters/bin/answer.py?hl=en&answer=156449
+# support.google.com/webmasters/answer/6062608
```
Comment (lyrixx) [inline web/robots.txt]: Let's use https?

Comment (Arman-Hosseini) [reply web/robots.txt]: Yes

Comment (lyrixx) [reply web/robots.txt]: Could you update the PR?
`https://support.google.com/webmasters/answer/6062608c`
thx

Comment (bocharsky-bw) [reply web/robots.txt]: Then I'd prefer to use HTTPS on the previous link as well, i.e. `https://www.robotstxt.org/`

**Review by Pierstoval — APPROVED** [review]

**Review by lyrixx — APPROVED** [review]

**Review by Tobion — APPROVED** [review]

**Review by colafati — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Pull request merged**